e2e/process-services/form-widgets-component.e2e.ts

```diff
@@ -100,63 +100,63 @@ describe('Form widgets', () => {
                     return formInstance;
                 })
                 .then(() => {
-                    expect(taskPage.formFields().getFieldLabel(appFields.text_id))
+                    expect(taskPage.formFields().getFieldLabel(appFields.text_id, undefined))
```

Comment:
wouldn't it be more elegant to have an optional parameter instead of using "undefined" ?

Reply:
I googled and came across this:
Optional and Default Parameters. In TypeScript, every parameter is assumed to be required by the function. This doesn't mean that it can't be given null or undefined , but rather, when the function is called the compiler will check that the user has provided a value for each parameter.
is it possible to add an optional parameter in typescript?

Reply:
Assuming you got this info from here: https://www.typescriptlang.org/docs/handbook/functions.html
The answer to your question is right below the fragment you copied:
`We can get this functionality in TypeScript by adding a ? to the end of parameters we want to be optional`

Reply:
great, will try that.